### PR TITLE
Marketplace Content Curation in Control Center 

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/developerportal/control-center/_index.md [hunk 1]
```diff
@@ -165,7 +165,29 @@ You can set a **Cover image** to be the background header of your company page.
...
 {{< figure src="/attachments/developerportal/control-center/brand.jpg" >}}
 
-## 8 Onboarding {#onboarding}
+## 8 Marketplace Content {#marketplace-content}
+
+The **Marketplace Content** page offers a dashboard where you can manage the visibility of public and private Marketplace components for Studio users from your company.
+
+{{< figure src="/attachments/developerportal/control-center/marketplace-content.png" >}}
+
+The list shows all the Marketplace components for which you have the curation options. This list includes all the public Marketplace components that can be used in Studio and all the private Marketplace components of your company. A Company Admin can configure whether any of these component is visible to the Studio Users from your company.
```
Comment (Adam-Dupaski) [inline content/en/docs/developerportal/control-center/_index.md]: "these component"

"Studio Users"

Comment (Adam-Dupaski) [reply content/en/docs/developerportal/control-center/_index.md]: Adding a cross-reference for "private content" would be useful: https://docs.mendix.com/appstore/general/app-store-overview/#company-content

"A Company Admin" > the assumed audience for this doc is a Company Admin, and it's specified at the top of the doc that only a Company Admin can see these pages, so I'd switch this to address the reader of the doc as "you"

Comment (Adam-Dupaski) [reply content/en/docs/developerportal/control-center/_index.md]: Actually the term "Company Admin" isn't even used anymore, it's "Mendix Admin," so this needs a proper update anyway

Comment (Luyao-Zhang-1) [reply content/en/docs/developerportal/control-center/_index.md]: Thanks for the review. I have incorporated the feedback to the doc!

**Pull request merged**